packages/postgres-database/src/simcore_postgres_database/migration/versions/ce69cc44246a_add_product_name_to_projects.py

```diff
@@ -0,0 +1,105 @@
+"""Add product_name to projects
+
+Revision ID: ce69cc44246a
+Revises: a85557c02d71
+Create Date: 2025-12-08 14:14:07.573764+00:00
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "ce69cc44246a"
+down_revision = "a85557c02d71"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column("projects", sa.Column("product_name", sa.String(), nullable=True))
+
+    op.execute(
+        """
+        UPDATE projects
```

Comment:
MINOR: this will eventually change the modified timestamp in the projects table?? I wonder if we could avoid it

Reply:
Well, as far as I know no. I don't see triggers.